core/backend/src/CloudSqlite.ts

```diff
@@ -360,8 +360,23 @@ export namespace CloudSqlite {
      * a 404 error. Default is 1 hour.
      */
     nSeconds?: number;
-    /** if enabled, outputs verbose logs about the cleanup process. These would include outputting blocks which are determined as eligible for deletion. */
+    /** if enabled, outputs verbose logs about the cleanup process. These would include outputting blocks which are determined as eligible for deletion.
+     * @default false
+    */
     debugLogging?: boolean;
+    /** If true, iterates over all blobs in the cloud container to add blocks that are 'orphaned' to the delete list in the manifest.
+     * Orphaned blocks are created when a client abruptly halts, is disconnected or encounters an error while uploading a change.
```

Comment:
```suggestion
     * Orphaned blocks are created when a client abruptly halts, is disconnected, or encounters an error while uploading a change.
```
(oxford comma nit :))